iceoryx_hoofs/source/posix_wrapper/message_queue.cpp

```diff
@@ -321,7 +321,9 @@ cxx::expected<IpcChannelError> MessageQueue::timedSend(const std::string& msg, c
 
 cxx::expected<bool, IpcChannelError> MessageQueue::isOutdated()
 {
-    struct stat sb;
+    struct stat sb
```

Comment:
Please write `stat sb{};` And if you are inspired by a better name let the inspiration directly flow into this piece of code :D